cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -380,6 +450,150 @@ writeLedgerState mOutFile qState@(SerialisedLedgerState serLedgerState) =
      first (const ls) (decodeFull ls)
 
 
+-- | This local data structure is introduced purely to allow nicely formatted output and has no other purpose.
+-- mark, set, go are the three ledger state stake snapshots (most recent to least recent)
+-- go is the snapshot that is used for the current epoch, set will be used in the next epoch,
+-- mark for the epoch after that.  marktot etc record the total active stake for each snapshot
+-- This information can be used by community tools to calculate upcoming leader schedules
+data Stakes =  Stakes {
```

Comment:
Let's move this type (along with its JSON instance) where it belongs, in `Cardano.CLI.Types`

Reply:
Done